qiime_studio/api/jobs.py

```diff
@@ -74,6 +74,15 @@ def create_job():
             parameters[key] = qiime.Metadata.load(
                 parameters[key][0]).get_category(parameters[key][1])
             json_params[key] = '<metadata>'
+        elif type_ == qiime.plugin.Bool:
+            # web forms exclude unchecked values, so we have to artificially
+            # "fill in the blanks" for the missing boolean parameters
+            try:
+                checked = parameters[key]
+            except:
+                checked = 'off'
+            parameters[key] = ('%s' % (checked == 'on')).lower()
+            json_params[key] = parameters[key]
```

Comment:
`qiime.core.type.primative.Bool.decode()` is looking for string literal `'true'/'false'`, so it fails passing actual booleans into it, is this expected? I moved handling to the client-side, but setting them to actual booleans in JS will translate to actual booleans server side `true -> True` and `false -> False`, so that's actually what that error I posted in #random was as well. Should I leave the JS side as strings instead of booleans, or add literals to the `Bool.decode()`?